.nvmrc

```diff
@@ -1 +1 @@
-10
+16
```

Comment:
is it a breaking change? 🤔

Reply:
je dirais que oui à cause du sort qui modifie l'ordre des choses

Reply:
je dirais que oui à cause du sort qui modifie l'ordre des choses

Reply:
> is it a breaking change? 🤔

No, only code used in production is subject to breaking change. If you add "engine: 16" to the package.json, then it's a breaking change. Here it's not. 

> je dirais que oui à cause du sort qui modifie l'ordre des choses

It doesn't change the order on our production code. It'll have no impact on app calling UI. So no, it's not a BC.